docs/contribute/manage-content.md

```diff
@@ -30,92 +30,69 @@ page_nav:
 
-<ul>
-<li>In content you can use an external links.</li>
-<li>On homepage header buttons you can use an external links but you have to set <code class="highlighter-rouge">external_url</code> to <code class="highlighter-rouge">true</code>. Same applies to author box on default and homepage layouts.</li>
-<li>In footer social list you <strong>have</strong> to use an external links.</li>
-</ul>
+### 404 error page
 
-<h2 id="relative-urls">Relative URLs</h2>
-<p>If you link internal pages in your content you have to add <code class="highlighter-rouge">baseurl</code> prefix to your links. This applies only to sites which will not be deployed in <code class="highlighter-rouge">root</code> directory of your server.</p>
+We have a custom 404 error page template. You can edit its content in `404.md`. If you’re hosting your site on Apache Web Servers, we’ve already added custom `.htaccess` file to update path to your 404 error page.
 
-<div class="callout callout--info">
-<p><strong>Relative URLs configuration.</strong> In "Configuration" section you can read more about relative URLs configuration.</p>
+>__Full 404 error page setup guide.__ <br> You can find full 404 error page setup guide [here](https://jekyllrb.com/tutorials/custom-404-page/).
+
+## External and internal links
+
+All links on your website should link to internal pages.  
+There’s few exception to this rule:
+
+*   In content you can use an external links.
+*   On homepage header buttons you can use an external links but you have to set `external_url` to `true`. Same applies to author box on default and homepage layouts.
+*   In footer social list you __have__ to use an external links.
+
+## Relative URLs
+
+If you link internal pages in your content you have to add `baseurl` prefix to your links. This applies only to sites which will not be deployed in `root` directory of your server.
+
+>__Relative URLs configuration.__ <br> In "Configuration" section you can read more about relative URLs configuration.
+
+## Syntax Highlighting
+
+You can call out code or a command within a sentence with single backticks. The text within the backticks will not be formatted.  
+To format code or text into its own distinct block, use triple backticks.  
+Best [example](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code) of this can be found [here](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+
+Furthermore, you can add an optional language identifier to enable syntax highlighting in your fenced code block. Best [example](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) of this can be found [here](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting).
+
+## Examples
+
+To add example bar add following `html` above your code block.
```

Comment:
I think we need an example of what this looks like, I'm not sure what an "example bar" is :confused:

Reply:
I also don't know how this looks like, so I implemented no changes regarding this - @IbbyBenali when you are free please let me know about this, or @ferrouswheel feel free to suggest an alternative meanwhile.